webpack.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    mode: "none",
+    entry: ["@babel/polyfill", "./lib/linter.js"],
+    output: {
+        filename: "eslint.js",
+        library: "eslint",
+        libraryTarget: "umd",
+        globalObject: "this"
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                loader: "babel-loader",
+                options: {
+                    presets: ["@babel/preset-env"]
```

Comment:
Using `@babel/preset-env` with no options compiles everything down to ES5, but we may not need to do that. Does someone have access to the site's Google Analytics to tell us what sort of minimum versions we need to support? Since our audience is mostly developers, there's a decent chance we'd only need to support the last couple versions of evergreen browsers and maybe the last IE.

[`@babel/preset-env` docs, for convenient reference.](https://babeljs.io/docs/en/babel-preset-env)

Reply:
Actually it was introduced in https://github.com/eslint/eslint/pull/11373, viewed as a dependency upgrade. seems we haven't got consensus on what sort of minimum versions we need to support. I'll create an issue in the site repo for discussion.

Reply:
Oh neat, thanks for linking that discussion.